### DISCUSSION TITLE
Cancel Animation Support for: Pointing, Crouching, Crawling

### DISCUSSION THREAD

**Discussion by khaosen** [alberttheprince/rpemotes-reborn#46] [Ideas]
When the cancel animation keybind is pressed/cancel animation command is used, if player uses any of these animations it cancels em as well.

**Comment by alberttheprince** [comment]
@MadsLeander is there a suggested way to account for this? I think you've made exports to detect if someone is crouching or crawling.

**Comment by iSentrie** [comment]
It doesn't cancel crouch/crawl, only pointing and that's how it has to be. It allows playing animations while crouched and it's a good feature.

**Comment by alberttheprince** [comment]
@theevelynnprivate I looked into it, seems to work fine on my side and doesn't cancel.
